snap/info.go

```diff
@@ -36,7 +36,10 @@ type PlaceInfo interface {
 	// Name returns the name of the snap.
 	Name() string
 
-	// MountDir returns the base directory of the snap.
+	// BaseDir returns the base directory for all versions of the snap.
+	BaseDir() string
```

Comment:
Agreed. This is part of why I used filepath.Clean initially-- this directory really has no meaning except for building paths. `ParentDir()`? `MountsDir()`? `MountParentDir()`?